migrations/staging/20171204143053_stg_arms_update_column_name.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex, Promise) {
+  var sql = 'EXEC sp_rename \'staging.arms.assessmentent_team_key\', \'assessment_team_key\';'
```

Comment:
Do you need to specify a third argument here, 'COLUMN'?
https://docs.microsoft.com/en-us/sql/relational-databases/system-stored-procedures/sp-rename-transact-sql

Reply:
The third parameter is for a data type change, we are not changing the type just the name.